src/address_operands.rs

```diff
@@ -176,11 +176,7 @@ fn address_operands(
                 ImmMemHandlerFlags::UseStackWithPushPop => {
                     // stack+=[src0 + offset] + src1
                     let src0 = reg_and_imm_write(vm, OutputOperandPosition::First, opcode);
-                    vm.current_frame_mut()?.sp += src0.value.low_u64() + 1;
-                    let sp = vm.current_frame()?.sp;
-                    vm.current_context_mut()?
-                        .stack
-                        .store_with_offset(1, res.0, sp)?;
+                    vm.current_frame_mut()?.sp += src0.value.low_u64();
```

Comment:
Stacks are 16-bits addressed, so I'm not sure `low_u64`  is enough bits dropped.

Reply:
Done